src/test/groovy/graphql/ScalarsQuerySchema.java

```diff
@@ -22,7 +22,7 @@ public class ScalarsQuerySchema {
             .field(newFieldDefinition()
                     .name("floatNaN")
                     .type(Scalars.GraphQLFloat)
-                    .staticValue(Double.NaN))
+                    .staticValue(Double.NaN)) // Retain for test coverage
```

Comment:
Isn't this one of the values we decided was not valid.

Reply:
Good catch, I'll include this in the NaN test tidy up. I can't see any test using this field at the moment